jans-config-api/plugins/admin-ui-plugin/pom.xml

```diff
@@ -10,20 +10,17 @@
     <groupId>io.jans.jans-config-api.plugins</groupId>
     <artifactId>admin-ui-plugin</artifactId>
 
-    <repositories>
-        <repository>
-            <id>jdk-java</id>
-            <url>https://licensespring-maven.s3.eu-central-1.amazonaws.com/</url>
-        </repository>
-    </repositories>
-
     <dependencies>
         <!-- config api -->
         <dependency>
             <groupId>io.jans</groupId>
             <artifactId>jans-config-api-shared</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jans</groupId>
+            <artifactId>jans-auth-client</artifactId>
+        </dependency>
```

Comment:
Sounds like with @pujavs changes we will get double declaration of the same dependency.

https://github.com/JanssenProject/jans/commit/8149133e635b83bb0d7a440ca9a5ff11d1e2edb7